livekit-agents/livekit/agents/worker.py

```diff
@@ -390,11 +391,23 @@ def run_app(worker: Worker) -> None:
     )
     @click.option(
         "--url",
-        help="The websocket URL",
-        default=worker._rtc_url,
+        required=True,
```

Comment:
required=True means it would fail for users who want to use `dotenv`

Reply:
No, it doesn't, because LIVEKIT_URL is read from dotenv before the agent start method is called.

Reply:
ahhh ok yeah!